modules/labs-react/combobox/spec/Combobox.spec.tsx

```diff
@@ -90,6 +90,18 @@ describe('Combobox', () => {
     });
   });
 
+  describe('when the child input is rendered with a ref', () => {
+    it('should set the ref to the child input element', async () => {
+      const ref: React.RefObject<HTMLInputElement> = React.createRef();
+      const {findByRole} = render(
+        <Combobox>
+          <TextInput ref={ref} />
+        </Combobox>
+      );
+      expect(await findByRole('combobox')).toEqual(ref.current);
+    });
```

Comment:
The rest of this file uses the `screen` interface. Also, why did you need to await? Would the following work?
```suggestion
    it('should set the ref to the child input element', () => {
      const ref: React.RefObject<HTMLInputElement> = React.createRef();
      render(
        <Combobox>
          <TextInput ref={ref} />
        </Combobox>
      );
      expect(screen.getByRole('combobox')).toEqual(ref.current);
    });
```

Reply:
@NicholasBoll The `async` and `await` are there from me copy-and-pasting [an old input ref test](https://github.com/Workday/canvas-kit/pull/1068/files#diff-00d14d17114b0b458110cb29abeca60151e11e4ff55e6b31522f84eeb9b9c688L58) and using it here. I thought we might be using `findByRole` due to `ref.current` not being set immediately, but the test seems to work fine if I remove `async/await` and use `getByRole` instead.

Regardless, I updated the test to use `screen`. Instead of importing `screen`, the Combobox tests set `screen` to the return value of `render` and run queries against it -- your code above works as long as I set `const screen = render(<Combobox>...</Combobox>)`.

Reply:
That's fine. React Testing Library uses `act` internally which synchronously flushes rendering updates. This is why `get*` works almost always. I really like the queries in React Testing Library, but honestly the differences between `get*`, `find*` and `query*` are kind of a mess. I didn't understand the differences until I started contributing to the project...